python/setup.py

```diff
@@ -84,8 +84,8 @@ def get_llvm_package_info():
         return Package("llvm", "LLVM-C.lib", "", "LLVM_INCLUDE_DIRS", "LLVM_LIBRARY_DIR", "LLVM_SYSPATH")
     use_assert_enabled_llvm = check_env_flag("TRITON_USE_ASSERT_ENABLED_LLVM", "False")
     release_suffix = "assert" if use_assert_enabled_llvm else "release"
-    name = f'llvm+mlir-17.0.0-{arch}-{system_suffix}-{release_suffix}'
-    version = "llvm-17.0.0-c5dede880d17"
+    name = f'llvm+mlir-18.0.0-{arch}-{system_suffix}-{release_suffix}'
+    version = "llvmorg-18-init-7000-g76ce4736721a"
```

Comment:
does this artifact exist? I don't see it here: https://github.com/ptillet/triton-llvm-releases/releases

Reply:
Ah no, forgot to ask: I don't know how to generate these!

Reply:
I kicked it off here, once the build ends and the PR is merged it should be ready to use:
https://github.com/ptillet/triton-llvm-releases/pull/18

Reply:
Seems like it merged, I rebased this PR and force-pushed, that should re-trigger the CI hopefully!

Reply:
actually I realized after that it starts a new build from scratch after the merge to create the tarball :(
https://github.com/ptillet/triton-llvm-releases/actions/runs/6330759872/job/17193877555

should take few more hours

Reply:
Saw this in the log: `#14 8927.6 [ 87%] Linking CXX executable ../../../../../bin/clang-ast-dump`

Do you know why we package clang here?

Reply:
ah yes I had added it due to old dependencies that happened during hopper transition (some helpers were compiled with clang). We should remove it now. I'll do it after this upgrade